.github/workflows/test.yml

```diff
@@ -27,6 +27,7 @@ jobs:
       run: pip install -e '.[all]'
     - name: Run Pytest
       run: |
+        LIGHTLY_SERVER_LOCATION="localhost:-1"
```

Comment:
what kind of evil python magic is this? ;) care to elaborate to a NAP?

Reply:
We're simply setting the environment variable to something which doesn't exist so the package is basically tested offline.